CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ and this project adheres to
 - Remove the following Node.js event listeners due to Node.js core deprecation:
   - `unhandledRejection`
   - `multipleResolves`
+- BREAKING: `executeWithContext(context: TExecutionContext, ...)` now requires
+  that `TExecutionContext.instance` is provided.
```

Comment:
Is there any way to avoid making this a breaking change? Is it possible to make `instance` an optional argument instead?